specificatie/openapi.yaml

```diff
@@ -182,12 +182,33 @@ components:
           $ref: '#/components/schemas/PersoonTypeEnum'
         vestigingsnummer:
           type: string
-    PersoonTypeEnum:
-      type: string
-      enum:
-        - "natuurlijk_persoon"
-        - "niet_natuurlijk_persoon"
-        - "vestiging"
+    KadastraleAanduiding:
+      type: object
+      properties:
+        gemeentecode:
+          description: Waar het kadaster de kadastrale gemeentenaam hanteert in de KadstraleAanduiding is in de WOZ de gemeentecode gebruikt.
+          type: string
+        sectie:
+          description: Sectie is een onderverdeling van de kadastrale gemeente, bedoeld om het werk van de meetdienst en de kadastrale kaarten overzichtelijk te houden. Per kadastrale gemeente zijn er max. 26x26 secties . Elke sectie heeft een of twee letters (bijv. "A", 'B', ....'AA', 'AB', ....'ZZ' . Alleen de sectieletter 'J' wordt niet gebruikt om verwarring (handgeschreven) te voorkomen met 'I'
+          type: string
+        perceelnummer:
+          description: Het perceelnummer dat een geheel perceel of een complex uniek identificeert binnen de sectie.
+          type: integer
+        deelperceelnummer:
```

Comment:
Was het niet zo dat er ofwel een deelperceelnummer ofwel een  appartementsindex aanwezig was? Zo ja, dan zou ik dat in de description van beide properties aangeven.